src/main/webapp/view/discussion/discussion.js

```diff
@@ -55,31 +58,40 @@ async function intializeDiscussion() {
  * Posts a new comment using the main post textarea.
  */
 async function postNewComment() {
-  postAndReload(
-      ELEMENT_POST_TEXTAREA, /* parent= */ undefined, newCommentTimestampMs);
+  // TODO: Add support for submitting types other than QUESTION.
+  postAndReload(ELEMENT_POST_TEXTAREA, {
+    [PARAM_TIMESTAMP]: newCommentTimestampMs,
+    [PARAM_TYPE]: COMMENT_TYPE_QUESTION,
```

Comment:
Can root comments be notes?

Reply:
They can be notes or questions.  The default here doesn't matter right now because it isn't displayed anywhere.